pom.xml

```diff
@@ -93,71 +186,6 @@
         <target.name>linux-aarch64</target.name>
       </properties>
     </profile>
-    <profile>
```

Comment:
Can Dapeng or anyone review and confirm that the "release" profile no longer needed?